AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+loading.com.br##.elementor-popup-modal
```

Comment:
This rule hides not only cookie notice but also another elements. For example it hides content after clicking on a menu button.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/108496782-d6484780-72aa-11eb-9e37-40c864d2441b.png)

</details>

I will change it to something more specific.